src/main/scala/viper/gobra/frontend/info/implementation/typing/TypeTyping.scala

```diff
@@ -101,6 +111,11 @@ trait TypeTyping extends BaseTyping { this: TypeInfoImpl =>
     case t: PType => typeType(t)
   }
 
+  lazy val isIntegerType: Type => Boolean = {
+    case IntT(_) => true
+    case _ => false
+  }
+
```

Comment:
This is the same as just an instanceOf[IntT]

Reply:
Also, it should be function and not a lazy field.

Reply:
removed `isIntegerType` in 8cb139e